src/libostree/ostree-impl-system-generator.c

```diff
@@ -93,18 +93,40 @@ path_kill_slashes (char *path)
 static char *
 stateroot_from_ostree_cmdline (const char *ostree_cmdline, GError **error)
```

Comment:
I'd like to do this in a follow on PR, but maybe these non-public functions should be like:

```
static char *
stateroot_from_ostree_target (const char *ostree_target, GError **error)
```

with:

s/ostree_cmdline/ostree_target/g

because in the aboot case for example, this parameter is not part of the cmdline, we get the ostree_target via other means.